src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -399,7 +400,7 @@ export default class SummaryPanelNode extends React.Component<SummaryPanelPropTy
       return (
         <>
           <div>
-            <Icon type="pf" name="info" /> Sparkline charts not supported for unknown node. Use edge for details.
+            <KialiIcon.Info /> Sparkline charts not supported for unknown node. Use edge for details.
```

Comment:
The solution is good, should this PR now apply the change everywhere, or is that a follow-up?

Reply:
I was thinking as a follow-up.